### PR TITLE
refactor!: remove tui-wrapper, tui-group, table-mode, field-error component, tui-breadcrumb items

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: projects/kit/components/breadcrumbs/breadcrumbs.template.html [hunk 1]
```diff
@@ -1,26 +1,9 @@
-<ng-container *ngFor="let item of items; last as lastItem">
-    <ng-container *tuiLet="item | tuiMapper: routerLinkOptions as options">
```
Comment (nsbarsukov) [inline projects/kit/components/breadcrumbs/breadcrumbs.template.html:2]: Should we also delete imports of `TuiMapperPipeModule` / `TuiLetModule` / `TuiLinkModule` / `RouterModule`  from the module of the component?

I am not sure that we need to delete all of them. May be some ones are redudant.

**Review by nsbarsukov — APPROVED** [review]

File: projects/kit/components/breadcrumbs/breadcrumbs.module.ts [hunk 2]
```diff
@@ -1,28 +1,12 @@
...
-        TuiMapperPipeModule,
-        TuiLetModule,
-        TuiLinkModule,
-        TuiSvgModule,
-        TuiModeModule,
-        TuiItemModule,
-    ],
+    imports: [CommonModule, TuiSvgModule],
     declarations: [TuiBreadcrumbsComponent],
-    exports: [TuiBreadcrumbsComponent, TuiItemDirective],
```
Comment (nsbarsukov) [inline projects/kit/components/breadcrumbs/breadcrumbs.module.ts]: no, dont delete it)

The idea of this export - user should not worry about importing of `TuiItemModule`

File: projects/kit/components/breadcrumbs/breadcrumbs.template.html (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/kit/components/breadcrumbs/breadcrumbs.template.html:2]: yes, thanks!

File: projects/kit/components/breadcrumbs/breadcrumbs.module.ts (hunk 2 above)
Comment (vladimirpotekhin) [reply projects/kit/components/breadcrumbs/breadcrumbs.module.ts]: you are right)

**Pull request merged**